src/signals/incident-management/containers/Dashboard/components/BarChart/vega-config-bar-chart.ts

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+
+export const vegaConfigBarChart: any = {
+  $schema: 'https://vega.github.io/schema/vega-lite/v5.json',
+  description: 'A bar chart showing cases with their statusses',
```

Comment:
Not very descriptive. Maybe something with "showing number of incidents per ..."?

Reply:
Done, twice now because apparently copy-pasting is very hard. (sigh)